app.js

```diff
@@ -1,57 +1,11 @@
-const productsCenterDOM = document.querySelector(".products-center");
-
-// get all products
-
-const getProducts = async function () {
-  let response = await fetch("products.json");
-  let data = await response.json();
-  let results = "";
-
-  for (let index = 0; index < data.length; index++) {
-    results += `
-       <article class="product">
-          <div class="img-container">
-              <img
-                  src="${data[index].url}"
-                  alt="product"
-                  class="product-img"
-              />
-              <button class="bag-btn" data-id="1">
-                  <i class="fas fa-shopping-cart"></i>
-                  add to bag
-              </button>
-          </div>
-          <h3>${data[index].name}</h3>
-          <h4>$${data[index].price}</h4>
-      </article>
-  `;
+let removeItemDOM = document.querySelectorAll(".remove-item");
+let cartItemDOM = document.querySelectorAll(".cart-item");
+
+// Implement Remove items button
+for (let i = 0; i < removeItemDOM.length; i++) {
+  removeItemDOM[i].addEventListener("click", removeItem);
+  function removeItem() {
+    cartItemDOM[i].style.visibility = "hidden";
```

Comment:
@SabaMallah if you hide the item, it's actually not removed but just hidden. Please remove it so that it's competely removed from the DOM and the items under can take their place otherwise, the place will still be reserved if you just hide the item.